### PR TITLE
Add Dan Winship as SIG Network Technical Lead

### PR DESCRIPTION
**Pull request by shaneutt**
**Which issue(s) this PR fixes**:

Part of resolving #7572.

/cc @thockin @MikeZappa87 @danwinship

### CODE REVIEW

File: sig-network/README.md [hunk 1]
```diff
@@ -42,6 +42,12 @@ The Chairs of the SIG run operations and processes governing the SIG.
 * Shane Utt (**[@shaneutt](https://github.com/shaneutt)**), Kong
 * Tim Hockin (**[@thockin](https://github.com/thockin)**), Google
 
+### Technical Leads
+The Technical Leads of the SIG establish new subprojects, decommission existing
+subprojects, and resolve cross-subproject technical issues and decisions.
+
+* Dan Winship (**[@danwinship](https://github.com/danwinship)**), Red Hat
```
Comment (thockin) [inline sig-network/README.md:49]: If we're adding this section, I suppose we should put me in there to be explicit.

File: sigs.yaml [hunk 2]
```diff
@@ -2120,6 +2120,10 @@ sigs:
     - github: thockin
       name: Tim Hockin
       company: Google
+    tech_leads:
```
Comment (thockin) [inline sigs.yaml:2123]: ditto

File: sig-network/README.md (hunk 1 above)
Comment (shaneutt) [reply sig-network/README.md:49]: sg 24be06e84480496e6c281c95bc2f7c63cc142fa8 :+1:

File: sigs.yaml (hunk 2 above)
Comment (shaneutt) [reply sigs.yaml:2123]: 24be06e84480496e6c281c95bc2f7c63cc142fa8 :+1:

**Comment by shaneutt** [comment]
@cblecker checking in?

**Comment by thockin** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**